### PR TITLE
Nicknames

### PR DESCRIPTION
**Pull request by ttbowen**
- Added a new command `username` for setting the user's nickname
- Updated the command emitter to search for command aliases

### CODE REVIEW

File: client/src/commands/index.ts [hunk 1]
```diff
@@ -2,60 +2,69 @@ import {
...
 } from "./help"
+import Nick from "./username"
 
-export const commandModules = [
-  Roll,
-  MakeRoom,
-  Go,
-  Help,
-]
+export const commandModules = [Roll, MakeRoom, Go, Help, Nick]
```
Comment (Andr3wRiv3rs) [inline client/src/commands/index.ts:28]: I think this should remain multiline as we'll eventually have loads of commands. This helps cut down on merge conflicts.

File: events.ts [hunk 2]
```diff
@@ -8,6 +8,7 @@ export const ROOM_UPDATE_EVENT = "ROOM_UPDATE"
 export const ROLL_EVENT = "ROLL"
 export const MAKE_ROOM_EVENT = "MAKE_ROOM"
 export const GO_EVENT = "GO"
+export const USERNAME_CHANGE_EVENT = "USERNAME_CHANGE_EVENT"
```
Comment (Andr3wRiv3rs) [inline events.ts:11]: Maybe we should suffix all events with `_EVENT` to match?

File: server/services/player.ts [hunk 3]
```diff
@@ -7,17 +7,19 @@ import {
...
-  const hash = crypto.createHash('md5')
+  const hash = crypto.createHash("md5")
   hash.update(publicKey)
-  
+
   const player: Player = {
     id: store.players.length,
     publicKey,
-    username: `1_${hash.digest('hex').slice(0, 6)}`,
+    username: `1_${hash.digest("hex").slice(0, 6)}`,
```
Comment (Andr3wRiv3rs) [inline server/services/player.ts:22]: Feel free to add an eslint rule for quotes. Double or single are okay.

**Review by Andr3wRiv3rs — CHANGES_REQUESTED** [review]

File: server/services/player.ts (hunk 3 above)
Comment (ttbowen) [reply server/services/player.ts:22]: I'd say double because they're used almost everywhere else in this project.

**Review by Andr3wRiv3rs — APPROVED** [review]

**Pull request merged**